tuxemon/item/conditions/has_path.py

```diff
@@ -44,6 +44,7 @@ class HasPathCondition(ItemCondition):
     expected: str
 
     def test(self, target: Monster) -> bool:
-        expect = self.expected
-
-        return any(d["path"] == expect for d in target.evolutions)
+        for evolution in target.evolutions:
+            if evolution.path.item:
+                if evolution.item == self.expected:
+                    return True
```

Comment:
Technically the signature is wrong because it returns true or none, not true or false.

Add return False after the for loop.

Reply:
True

Reply:
You added it within the for loop